app/uk/gov/hmrc/eoricommoncomponent/frontend/models/enrolmentRequest/KnownFactsQuery.scala

```diff
@@ -25,7 +25,7 @@ object KnownFactsQuery {
 
   def apply(eoriNumber: String): KnownFactsQuery =
     new KnownFactsQuery(
-      service = "HMRC-CUS-ORG",
+      service = "HMRC-CUS-ORG",  // TODO check - should this be [SERVICE]
```

Comment:
KnownFacts query will always need to be HMRC-CUS-ORG.

Autoenrolment can be based only on CDS enrolment. So we will be always retrieving only known facts for CDS.

Reply:
That makes sense.  Thanks.  I'll remove the TODO here.